CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## PENDING
```

Comment:
Can make this 4.2.0. package.json is already bumped, will be a normal release from master

Reply:
My inclination is to leave this as PENDING until we're ready for release, as it helps with people on github asking why the version on NPM doesn't agree with what they are seeing, but I'm open to persuasion.